sound/soc/intel/common/soc-acpi-intel-lnl-match.c

```diff
@@ -93,6 +93,30 @@ static const struct snd_soc_acpi_endpoint jack_amp_dmic_endpoints[] = {
 	},
 };
 
+static const struct snd_soc_acpi_endpoint jack_amp_g1_dmic_endpoints_endpoints[] = {
```

Comment:
I cannot see where you;re reusing jack_amp_dmic_endpoints that you renamed in the first commit @bardliao

Reply:
then do you want to drop it for now. It doesnt make sense in this PR and the commit specifically says you are going to reuse in the following commit and then you ont

Reply:
Yes, dropped.